### ISSUE TITLE
NVDA Reads Column Header Twice

### ISSUE THREAD

**Issue by magleaso** [nvaccess/nvda#10177]
### Steps to reproduce:
1. Open Chrome
2. Navigate to [an Azure DevOps Feed](https://dev.azure.com/azure-public/vside/_packaging?_a=feed&feed=vssdk)
3. Tab through the table
### Actual behavior:
Column names are read twice
### Expected behavior:
Column names are read only once
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Version 2019.1
#### Windows version:
Windows 10 Version 0.18362 Build 18362
#### Name and version of other software in use when reproducing the issue:
Chrome 76
#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, version 2019.2 has same behavior

**Comment by magleaso** [comment]
Narrator does not exhibit the same behavior. When the cellTitle is removed from the class property of the header, NVDA stops double reading.
![html tag with highlighted cell title segment. The tag reads: <span id="header7-location" aria-labelledby="header7-location-name " class="ms-DetailsHeader-cellTitle *begin highlight* cellTitle_b85d5fbe*end highlight*" data-is-focusable="false" aria-haspopup="false" tabindex="-1">](https://user-images.githubusercontent.com/7018576/64559906-35efa480-d2fc-11e9-935b-94df52340edc.png)

**Comment by magleaso** [comment]
Perhaps this could be folded up into #10140

**Comment by Adriani90** [comment]
Closing as duplicate of #10840 which can be reproduced more easily
